matrix_benchmarking/visualize.py

```diff
@@ -55,7 +55,7 @@ def run():
         # ---
 
         logging.info(f"Loading results ... ")
-        workload_store.parse_data()
+        workload_store.parse_data(lts=kwargs.get("lts"))
```

Comment:
Ok sounds good! This might be a mistake on my part during the rebase. I might have resolved the conflict between the versions of `parse_data` incorrectly!

Reply:
hum, true indeed :) if you look at the commits of the PR, I am part of 2 of them. These two are "mistakes" from your rebase, and should be deleted